app/services/dttp/defer.rb

```diff
@@ -14,9 +14,13 @@ def initialize(trainee:)
     end
 
     def call
-      body = { "dfe_TraineeStatusId@odata.bind" => "/dfe_traineestatuses(#{dttp_status_id('Trainee deferred')})" }
-
-      dttp_update("/dfe_placementassignments(#{trainee.placement_assignment_dttp_id})", body)
+      dttp_update(
```

Comment:
We can remove `include Mappable` in this file now we're not using the helper method `dttp_status_id ` directly.